strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -154,8 +156,15 @@ protected boolean isChecksum(String fileName)
                                                               RepositoryFileAttributeType.ARTIFACT);
 
                 boolean isArtifact = Boolean.TRUE.equals(attributesLocal.get(RepositoryFileAttributeType.ARTIFACT));
-
-                value = isArtifact ? getArtifactCoordinates(repositoryPath) : null;
+                if (!isArtifact) {
+                    value = null;
+                    break;
+                }
```

Comment:
Cosmetic: Braces on new lines please. :)
```suggestion
                if (!isArtifact)
                {
                    value = null;
                    break;
                }
```